# the master func hz
loopRate: 10

# parameter about searching algorithm
searchLightRadius: 0.5
searchLightRange: 1.047
inscribedRadius: 0.2

#multi master msg rate
multiMasterMsgRate: 50.0
teleopNodeUri: http://teleopNode:11311
teleopNodePubName: /teleop_node/flag_from_master
teleopNodePubType: std_msgs/Bool
teleopNodeSubName: /teleop_node/pose_pub
teleopNodeSubType: geometry_msgs/PoseStamped

npc1NodeUri: http://157.82.5.92:11311
npc1NodePubName: /npc1_node/flag_from_master
npc1NodePubType: std_msgs/Bool
npc1NodeSubName: /npc1_node/pose_pub
npc1NodeSubType: geometry_msgs/PoseStamped

npc2NodeUri: http://157.82.4.160:11311
npc2NodePubName: /npc2_node/flag_from_master
npc2NodePubType: std_msgs/Bool
npc2NodeSubName: /npc2_node/pose_pub
npc2NodeSubType: geometry_msgs/PoseStamped

